lib/internal/process/pre_execution.js

```diff
@@ -61,7 +61,7 @@ function prepareMainThreadExecution(expandArgv1 = false, initializeModules = tru
 function prepareWorkerThreadExecution() {
   prepareExecution({
     expandArgv1: false,
-    initializeModules: false,  // Will need to initialize it after policy setup
+    initializeModules: false,
```

Comment:
I have looked at the code, but I couldn't see a place where this value gets changed (regardless of policy is enabled)

Reply:
This still cannot be true because:

1. In the worker, we can only initialize the module loader after we get the message from the main thread and know that the worker is spawned for. If it's spawned for the loader worker, then it needs to be initialized differently to avoid spawning another loader worker from the loader worker and we run into the infinite loader worker bug.
2. We probably are not in a shape to execute preloaded modules at the beginning of the worker bootstrap (when `prepareWorkerThreadExecution()` is called) and can only do it after we [do all these](https://github.com/nodejs/node/blob/075c95f61f2be5b8571fe4c6fcf9c3744d0188a9/lib/internal/main/worker_thread.js#L102-L131) even minus policy setup.